src/solvers/petsc_dm_wrapper.C

```diff
@@ -53,7 +56,7 @@ namespace libMesh
 #if PETSC_VERSION_LESS_THAN(3,9,0)
     PetscErrorCode libmesh_petsc_DMCreateSubDM(DM dm, PetscInt numFields, PetscInt fields[], IS *is, DM *subdm)
 #else
-    PetscErrorCode libmesh_petsc_DMCreateSubDM(DM dm, PetscInt numFields, const PetscInt fields[], IS *is, DM *subdm)
+      PetscErrorCode libmesh_petsc_DMCreateSubDM(DM dm, PetscInt numFields, const PetscInt fields[], IS *is, DM *subdm)
```

Comment:
Remove extra space?

Reply:
Good catch! Looks like I need to check my emacs config.

Reply:
This is just emacs being confused about the correct indentation level because of surrounding ifdefs... I have seen it before.